roles/ideascube/tasks/ideascube.yml

```diff
@@ -191,3 +191,15 @@
     and is_upgraded.changed == True
   tags:
     - update
+
+- name: Copy ideascube_leftover_files.service systemd unit
+  copy: src=ideascube_leftover_files.service dest=/etc/systemd/system/ideascube_leftover_files.service
+  tags: ['master', 'custom', 'update']
+
+- name: Copy ideascube_leftover_files.timer systemd unit
+  copy: src=ideascube_leftover_files.timer dest=/etc/systemd/system/ideascube_leftover_files.timer
```

Comment:
I wonder if we should add these 2 files to the Ideascube package. 😄 

Maybe let's keep them in Ansiblecube as it makes it easier to experiment, and if we're happy with them we can decide to move them later.